src/menu/ide_menu.rs

```diff
@@ -1487,4 +1493,50 @@ mod tests {
         menu.update_values(&mut editor, &mut completer);
         assert!(menu.menu_string(2, true).contains("おは"));
     }
+
+    #[test]
+    fn test_menu_create_value_string_starting_with_multibyte_char() {
+        // https://github.com/nushell/nushell/issues/15938
+        let mut completer = FakeCompleter::new(&["验abcdef/", "abcdef/"]);
+        let mut menu = IdeMenu::default().with_name("testmenu");
+        menu.working_details = IdeMenuDetails {
+            cursor_col: 50,
+            menu_width: 50,
+            completion_width: 50,
+            description_width: 50,
+            description_is_right: true,
+            space_left: 50,
+            space_right: 50,
+            description_offset: 50,
+            shortest_base_string: String::new(),
+        };
+        let mut editor = Editor::default();
+
+        editor.set_buffer("ac".to_string(), UndoBehavior::CreateUndoPoint);
+        menu.update_values(&mut editor, &mut completer);
+        assert!(menu.menu_string(10, true).contains("验"));
+    }
+
+    #[test]
+    fn test_menu_create_value_string_long_unicode_string() {
+        // https://github.com/nushell/reedline/pull/918
+        let mut completer = FakeCompleter::new(&["验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验验abcdef/", "abcdef/"]);
```

Comment:
Is the second "abcdef/" suggestion necessary? I'd just do something like this:
```suggestion
        let mut completer = FakeCompleter::new(&["验验验验abcdef"]);
```

Reply:
The second abcdef/ came from when testing it live in an actual folder, you need two files or the menu won't even appear. But you're right I just tested it here and FakeCompleter runs the menu even with one entry.

I've reduced it to just `FakeCompleter::new(&["验abc/"])` as that's all that's needed